src/macpro-security-hub-sync.ts

```diff
@@ -176,7 +176,27 @@ export class SecurityHubJiraSync {
     Table += `------------------------------------------------------------------------------------------------`;
     return Table;
   }
+  createSecurityHubFindingUrlThroughFilters(findingId: string) {
+    let region, accountId;
+
+    if (findingId.startsWith("arn:")) {
+      // Extract region and account ID from the ARN
+      const arnParts = findingId.split(":");
+      region = arnParts[3];
```

Comment:
Would it make sense to check if the split is a success here before assigning value? That way an unexpected format doesn't throw an error.

Reply:
Agreed, I have updated the code